.github/guidelines/LABELING_GUIDELINES.md

```diff
@@ -1,6 +1,12 @@
 # PR Labeling Guidelines
 To maintain a consistent and efficient development workflow, we have set specific label guidelines for all pull requests (PRs). Please ensure you adhere to the following instructions:
 
+### PR Titles:
```

Comment:
This comment should probably go in a different file, as it is not about a PR label, but a PR title.

Perhaps we should create a new `PR_GUIDELINES.md`  file in the same directory, or add a file in the `docs/` directory

Reply:
Actually, adding it to the `CONTRIBUTING.md` file probably makes the most sense

Reply:
Moved it to `CONTRIBUTING.md`, and also made several other changes to `CONTRIBUTING.md`:
- deleted `Add to the CHANGELOG` -- we're not doing this anymore
- `Close the issue` -- edited to match the new PR template
- `Get reviewed by a core contributor` -- edited to `**Get reviewed by MetaMask Internal Developers**: All PRs require 2 approvals from MetaMask Internal Developers before merging.`
- `LABELING_GUIDELINES` -- fixed the broken hyperlink